README.md

```diff
@@ -1,6 +1,6 @@
-# Node.js Long-term Support Working Group
+# Node.js Release and Long-term Support Working Group
```

Comment:
It feels like maintaining the LTS releases really falls under the greater umbrella of maintaining releases in general, so the WG could just be called _Release WG_.